roles/ssl_certificates/tasks/manual.yml

```diff
@@ -0,0 +1,24 @@
+- name: Copy manual SSL certificate
+  copy:
+    src: "{{ site_ssl.manual.certificate }}"
+    dest: "{{ nginx_ssl_path }}/{{ site_ssl.manual.certificate | basename }}"
+    mode: '0640'
+  with_dict: "{{ wordpress_sites }}"
+  notify: reload nginx
+
+- name: Copy manual SSL certificate key
+  copy:
+    src: "{{ site_ssl.manual.key }}"
+    dest: "{{ nginx_ssl_path }}/{{ site_ssl.manual.key | basename }}"
+    mode: '0600'
+  with_dict: "{{ wordpress_sites }}"
+  notify: reload nginx
+
+- name: Download SSL client certificate
```

Comment:
Client certificate shouldn't be "scoped" under `manual`.
Client certificates are independent from ssl providers.

E.g: Client certificates can be used with `acme`, `manual` and other 3rd-party providers.

Reply:
Oh, thanks! I've moved this task into `main.yml`.